manifests/g/GHannesDE/FoxisWorld/1.1/GHannesDE.FoxisWorld.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: GHannesDE.FoxisWorld
+PackageVersion: 1.1
+PackageLocale: en-US
+Publisher: GHannesDE
+PackageName: FoxisWorld
+License: none

```

Comment:
```
License: none
```
This needs to be changed (either MIT or something else), by default (depending on which country you live in) you're the only one that has the rights to the software and if there's no license for it, no one can use your software without your permission.